models/experimental/mamba/tests/test_full_model.py

```diff
@@ -56,7 +56,7 @@ def test_mamba_model_inference(
     model_version: MambaPretrainedModelName,
     batch: int,
     pcc: float,
-    device: tt_lib.device,
```

Comment:
Not sure if this refers to the device or device module in tt_lib...
for now let's keep it ttnn.experimental.device, when getting rid of ttnn/tt_lib folder will change this.
@arakhmati will that be ok ?